tests/conftest.py

```diff
@@ -96,8 +104,17 @@ def mindmeld_project_path2():
 
 
 @pytest.fixture(scope="session")
-def dialogflow_project_path():
-    return DIALOGFLOW_PROJECT_PATH
+def mm_df_app_path():
+    return MM_DF_PATH
+
+
+@pytest.fixture(scope="session")
+def mm_df_nlp(mm_df_app_path):
+    """Provides a built processor instance"""
+    nlp = NaturalLanguageProcessor(app_path=mm_df_app_path)
+    nlp.build()
+    nlp.dump()
+    return nlp
```

Comment:
do these files get cleaned up after the test?

Reply:
done